### PR TITLE
Default lang argument in Tera templates

### PR DESCRIPTION
**Pull request by technic**
this makes lang argument optional by providing global default value

See https://github.com/XAMPPRocky/fluent-templates/issues/13

@XAMPPRocky is it what you had in mind?

### CODE REVIEW

**Comment by XAMPPRocky** [comment]
Thank you for your PR! This looks great to me. Would you also be able to add a test that makes sure it works and that tera errors if you don't provide a `lang` parameter *and* don't have a default language?

**Comment by technic** [comment]
Great! I will add a test for it.

File: templates/src/loader.rs [hunk 1]
```diff
@@ -80,12 +80,23 @@ where
...
     /// Create a new `FluentLoader`.
     pub fn new(loader: L) -> Self {
-        Self { loader }
+        Self {
+            loader,
+            default_lang: None,
+        }
+    }
+
+    pub fn with_default_lang(self, lang: LanguageIdentifier) -> Self {
```
Comment (XAMPPRocky) [inline templates/src/loader.rs:95]: Could also add a bit of documentation about the feature here?

Comment (technic) [reply templates/src/loader.rs:98]: Done

**Comment by XAMPPRocky** [comment]
Thank you for your PR, and congrats on your first contribution! 🎉

**Comment by XAMPPRocky** [comment]
This has now been published in 0.5.11

**Pull request merged**